geomstats/geometry/spd_matrices.py

```diff
@@ -367,7 +367,8 @@ def __init__(self, n, power_affine=1):
         self.space = SPDMatrices(n)
         self.power_affine = power_affine
 
-    def _aux_inner_product(self, tangent_vec_a, tangent_vec_b, inv_base_point):
+    @staticmethod
+    def _aux_inner_product(tangent_vec_a, tangent_vec_b, inv_base_point):
```

Comment:
What are these? Why inv_base_point?

Reply:
This was already here, I just added the @staticmethod decorator according to DeepSource

Reply:
inv stands for inverse.